.github/actions/gradle-cache/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Gradle Cache'
```

Comment:
Are we planning to extract these actions to a external repository to share it across repositories (Chat, Video, Push, ...)

Reply:
It is possible, yes.
Once this is merged and tested in the development workflow, we could think about that... 👍🏻